udm-le.sh

```diff
@@ -183,6 +184,18 @@ install_lego() {
 	fi
 }
 
+install_java() {
+	# Check if lego exists already, do nothing
+	if [ ! -f "${JAVA_BINARY}" ] || [ "${JAVA_FORCE_INSTALL}" = true ]; then
+		echo "install_java(): Attempting java installation"
+
+		# install jre via apt
+		apt install default-jre-headless
```

Comment:
I think you want this to be `apt install -y --no-install-recommends default-jre-headless` so it doesn't prompt, and so it doesn't drag in a bunch of other packages it doesn't need.

Also I think we might want to change line 189 to take into account if people don't have `ENABLE_CAPTIVE` set to `yes`, since they won't actually need java if they're not using the captive portal.

Does that make sense? I'd also ditch the else log line, that way people without ENABLE_CAPTIVE set won't be seeing a log line about java stuff when they don't even need it.

Reply:
1. @kchristensen Yup ... obvious mistake on my side. 
2. Yep, makes sense.

Fixing in a bit.